sentry/src/main/java/io/sentry/ISpan.java

```diff
@@ -169,4 +169,27 @@ ISpan startChild(
    */
   @Nullable
   Object getData(@NotNull String key);
+
+  /**
+   * Set a measurement with NONE unit.
```

Comment:
```suggestion
   * Set a measurement without unit.
```

In case we decide to no longer default to NONE in the SDK.